src/app.ts

```diff
@@ -79,7 +79,8 @@ export class CeramicAnchorApp {
 
     if (config.metrics.exporterEnabled) {
       try {
-        Metrics.start({ metricsPort: config.metrics.port, metricsExporterEnabled: true })
+        Metrics.start({ metricsPort: config.metrics.port, metricsExporterEnabled: true }, 'cas-' + config.mode)
+        Metrics.count('HELLO', 1) 
```

Comment:
Do we want this left in?

Reply:
yes - I want it there to see that metrics are happening at all, I have that in each node so that I can see the number of times the node starts